File: OSCommon/org/optimizationservices/oscommon/datastructure/osresult/ResultData.cs
using System;

using org.optimizationservices.oscommon.datastructure.osprocess;

namespace org.optimizationservices.oscommon.datastructure.osresult{
	/// <summary>
	/// A data structure class that corresponds to an xml element in the OSrL schema. 
	/// @author Jun Ma	
	/// @version 1.0, 09/01/2005
	/// @since OS 1.0
	/// @copyright (c) 2005
	/// </summary>
	public class ResultData{

		public OptimizationResult optimization = null;
		
		public ProcessStatistics statistics = null;
		
		public OtherResult[] other = null;

	}//class ResultData
}//namespace

File: OSCommon/bak/osresult_complex/ResultData.java
/**
 * @(#)ResultData 1.0 03/14/2004
 *
 * Copyright (c) 2004
 */
package org.optimizationservices.oscommon.datastructure.osresult;

import org.optimizationservices.oscommon.datastructure.osprocess.ProcessStatistics;

/**
*
* <p>A data structure class that corresponds to an xml element in the OSrL schema.  
*  
* </p>
* @author Robert Fourer, Jun Ma, Kipp Martin
* @version 1.0, 03/14/2004
* @since OS 1.0
*/
public class ResultData {

		public ProcessStatistics statistics = null;

		public OptimizationResult optimization = null;
			
		public OtherResult[] other = null;
		
}//ResultData
